sdk/go.mod

```diff
@@ -4,7 +4,6 @@ go 1.19
 
 require (
 	github.com/blang/semver v3.5.1+incompatible
-	github.com/pkg/errors v0.9.1
```

Comment:
This is a b it surprising, any idea why it moved from here to indirect requirements? (or indeed, how it managed to be here in the first place)